apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/template/TemplateController.java

```diff
@@ -0,0 +1,39 @@
+package gov.cdc.nbs.questionbank.template;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.cdc.nbs.questionbank.template.request.TemplateSearchRequest;
+import gov.cdc.nbs.questionbank.template.response.Template;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/api/v1/template/")
+@RequiredArgsConstructor
+@PreAuthorize("hasAuthority('LDFADMINISTRATION-SYSTEM')")
+public class TemplateController {
+
+	private TemplateReader templateReader;
+
+	@GetMapping
+	@PreAuthorize("hasAuthority('LDFADMINISTRATION-SYSTEM')")
```

Comment:
Can leave this `@PreAuthorize` off since it is applied at the class level

Reply:
@mpeels  updated